src/app/content/components/Toolbar/MyHighlights/HighlightsPopUp.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+import myHighlightsEmptyImage from '../../../../../assets/MHpage-empty-logged-in.png';
+import notLoggedImage1 from '../../../../../assets/My_Highlights_page_empty_1.png';
+import notLoggedImage2 from '../../../../../assets/My_Highlights_page_empty_2.png';
+import * as authSelect from '../../../../auth/selectors';
+import { User } from '../../../../auth/types';
+import * as selectNavigation from '../../../../navigation/selectors';
+import { AppState, Dispatch } from '../../../../types';
+import { closeMyHighlights } from '../../../highlights/actions';
+import * as selectors from '../../../highlights/selectors';
+import * as Styled from './HighlightStyles';
+
+interface Props {
+  myHighlightsOpen: boolean;
+  closeMyHighlights: () => void;
+  user?: User;
+  loggedOut: boolean;
+  currentPath: string;
+}
+
+class HighlightsPopUp extends Component<Props> {
+  public loginForHighlights = () => {
+    return <Styled.PopupBody>
+      <Styled.LoginText values={{loginLink: '/accounts/login?r=' + this.props.currentPath}} />
```

Comment:
i added `loginLink` to authSelectors so you don't have to have `'/accounts/login?r='` hardcoded here anymore